server/plugin/utils_test.go

```diff
@@ -659,7 +662,10 @@ func TestGetOffsetAndLimitFromQueryParams(t *testing.T) {
 		},
 	} {
 		t.Run(testCase.description, func(t *testing.T) {
-			mockAPI.On("LogError", mock.AnythingOfType("string"), mock.AnythingOfType("string"), mock.AnythingOfType("string"))
+			if testCase.queryParamPage != "1" && testCase.queryParamPerPage != "10" && testCase.expectedLimit != 10 && testCase.expectedOffset != 10 {
```

Comment:
There must be constants present for these values `10`.

Reply:
These are just some mock values